data/mods/gen6xy/moves.ts

```diff
@@ -0,0 +1,18 @@
+export const Moves: {[k: string]: ModdedMoveData} = {
+	dragonascent: {
+		inherit: true,
+		isNonstandard: "Future",
+	},
+	hyperspacefury: {
+		inherit: true,
+		isNonstandard: "Future",
+	},
+	originpulse: {
+		inherit: true,
+		isNonstandard: "Future",
+	},
+	precipiceblades: {
+		inherit: true,
+		isNonstandard: "Future",
+	},
+};
```

Comment:
I believe Thousand Arrows/Waves need to be here as Unobtainable so Smeargle can't use them?

Reply:
Both moves are already marked as `Unobtainable` in the base gen 6 mod